SSFWServer/SSFWServer.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net7.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
We moved to net7.0 on master repo

Reply:
I was trying to ensure this didn't slip for compatibility on my end.. because I am NOT arsed to remake a docker image just for a net7.0 change lol